content/working-with-azure-key-vault-in-csharp/index.md

```diff
@@ -410,7 +410,7 @@ services.AddControllersWithViews();
 }
 ```
 
-## **Deploy the Application to Azure**
+## Deploy the Application to Azure
 
 To deploy the application, follow the steps mentioned below:
```

Comment:
Are ellipses necessary here: Select Publish`...`

Reply:
Changes: Specify, or associate`,` the App Service instance with your application

Reply:
> 
> 
> Are ellipses necessary here: Select Publish`...`

No, I've removed it.